frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { OverlayTrigger, Tooltip } from 'patternfly-react';
+
+import { LoadingInline } from '../../utils';
+
+export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value, isLoading }) => {
+  const description = value ? (
+    <OverlayTrigger
+      overlay={<Tooltip id={`tooltip-for-${title}`}>{value}</Tooltip>}
```

Comment:
Are we sure this will be unique and a valid `id`? I'd suggest generating a value instead with `_.uniqueId`

Reply:
using `_.uniqueId` now